parquet-cli/src/test/resources/cli-output/show_footer_command.txt

```diff
@@ -0,0 +1,331 @@
+        "length" : 0,
+        "decimalMeta" : null,
+        "columnOrder" : {
+          "columnOrderName" : "TYPE_DEFINED_ORDER"
+        }
+      }, {
+        "name" : "double_field",
+        "repetition" : "REQUIRED",
+        "logicalTypeAnnotation" : null,
+        "id" : null,
+        "primitive" : "DOUBLE",
+        "length" : 0,
+        "decimalMeta" : null,
+        "columnOrder" : {
+          "columnOrderName" : "TYPE_DEFINED_ORDER"
+        }
+      }, {
+        "name" : "binary_field",
+        "repetition" : "REQUIRED",
+        "logicalTypeAnnotation" : null,
+        "id" : null,
+        "primitive" : "BINARY",
+        "length" : 0,
+        "decimalMeta" : null,
+        "columnOrder" : {
+          "columnOrderName" : "TYPE_DEFINED_ORDER"
+        }
+      }, {
+        "name" : "flba_field",
+        "repetition" : "REQUIRED",
+        "logicalTypeAnnotation" : null,
+        "id" : null,
+        "primitive" : "FIXED_LEN_BYTE_ARRAY",
+        "length" : 12,
+        "decimalMeta" : null,
+        "columnOrder" : {
+          "columnOrderName" : "TYPE_DEFINED_ORDER"
+        }
+      }, {
+        "name" : "date_field",
+        "repetition" : "REQUIRED",
+        "logicalTypeAnnotation" : { },
+        "id" : null,
+        "primitive" : "INT32",
+        "length" : 0,
+        "decimalMeta" : null,
+        "columnOrder" : {
+          "columnOrderName" : "TYPE_DEFINED_ORDER"
+        }
+      } ],
+      "indexByName" : {
+        "binary_field" : 4,
+        "date_field" : 6,
+        "double_field" : 3,
+        "flba_field" : 5,
+        "float_field" : 2,
+        "int32_field" : 0,
+        "int64_field" : 1
+      }
+    },
+    "keyValueMetaData" : {
+      "writer.model.name" : "example"
+    },
+    "createdBy" : "*****",
+    "encryptionType" : "UNENCRYPTED"
+  },
+  "blocks" : [ {
+    "columns" : [ {
+      "rowGroupOrdinal" : 0,
+      "encodingStats" : {
+        "dictStats" : { },
+        "dataStats" : {
+          "DELTA_BINARY_PACKED" : 1
+        },
+        "usesV2Pages" : true
+      },
+      "properties" : {
+        "codec" : "UNCOMPRESSED",
+        "path" : {
+          "p" : [ "int32_field" ]
+        },
+        "encodings" : [ "DELTA_BINARY_PACKED" ]
+      },
+      "columnIndexReference" : {
+        "offset" : 573,
+        "length" : 23
+      },
+      "offsetIndexReference" : {
+        "offset" : 768,
+        "length" : 10
+      },
+      "bloomFilterOffset" : -1,
+      "bloomFilterLength" : -1,
+      "firstDataPage" : -2147483644,
+      "dictionaryPageOffset" : -2147483648,
+      "valueCount" : -2147483638,
+      "totalSize" : -2147483610,
+      "totalUncompressedSize" : -2147483610
+    }, {
+      "rowGroupOrdinal" : 0,
+      "encodingStats" : {
+        "dictStats" : { },
+        "dataStats" : {
+          "DELTA_BINARY_PACKED" : 1
+        },
+        "usesV2Pages" : true
+      },
+      "properties" : {
+        "codec" : "UNCOMPRESSED",
```

Comment:
this test uses UNCOMPRESSED thus the size of data SHOULD be platform-agnostic, while if it uses codec like zstd, gzip, the compressed size might be different in different platforms, let's say I use macOS M1 machine to produce the golden file, which might be different from the output of Linux x64 machine.